creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -288,17 +288,19 @@ const _determineNewReplicaSet = async ({
 }: DetermineNewReplicaSetParams) => {
   const currentReplicaSet = [primary, secondary1, secondary2]
   const healthyReplicaSet = new Set(
-    currentReplicaSet.filter((node) => !unhealthyReplicasSet.has(node))
+    currentReplicaSet.filter((node) => node && !unhealthyReplicasSet.has(node))
```

Comment:
we shouldn't count a node as healthy if it's empty. context here is that findReplicaSetUpdates jobs will just ignore empty/falsy nodes so they won't flag them as unhealthy either